frontend/src/routes/_dashboard.tsx

```diff
@@ -100,7 +100,9 @@ function Header() {
                 <ChevronsUpDown className="w-4" />
               </MenubarTrigger>
               <MenubarContent className=" border border-border min-w-6">
-                <MenubarContentItem icon={Folder} text="Project" />
+                <a href="/create-project">
```

Comment:
Why not use a `<Link>` component here ? This will reload the page instead of navigating client side